eda-modbus-bridge/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
   * Configure entity icons for alarm sensors
   * Move Home Assistant related logic to `homeassistant.mjs`
   * Publish device information over MQTT once only (https://github.com/Jalle19/eda-modbus-bridge/issues/43)
+- Change default publish interval to 60s, see [DOCS.md](https://github.com/Jalle19/home-assistant-addon-repository/blob/main/eda-modbus-bridge/DOCS.md) for more info (https://github.com/Jalle19/home-assistant-addon-repository/issues/14)
```

Comment:
I pulled the trigger on the release already, can you move this to a new 1.2.1 release?

Reply:
I noted that myself too, but, as this addon is not released as such, any change in the main branch comes out HA aswell, right? This mean my change in itself here would be a "release". Should I change all the versioning aswell? In other words we will get 1.2.1 available once merged?

Reply:
Increased version already.

Reply:
Yeah I think it's best to make a new release either way, the Dockerfile pulls a specific tag so it's not enough to just push something to master.

Reply:
From the eda-modbus-bridge repo yes. But this repo has no tags, HA pulls latest only.

Reply:
Yeah